devops/github_issue_alert/program.py

```diff
@@ -0,0 +1,38 @@
+"""Monitors GitHub issue creation and comment activity.
+
+Sending the notifications to Slack to keep teams updated.
+"""
+
+import os
+
+from autokitteh.slack import slack_client
+
+
+SLACK_CHANNEL = os.getenv("SLACK_CHANNEL", "")
+
+slack = slack_client("slack_conn")
+
+
+def on_issue_comment(event):
+    comment_body = event["data"]["comment"]["body"]
+    comment_action = event["data"]["action"]
+    comment_author = event["data"]["comment"]["user"]["login"] or "Unknown user"
+    issue_title = event["data"]["issue"]["title"]
+
+    text = (
+        f"{comment_author} {comment_action} a comment: '{comment_body}' "
+        f"on issue '{issue_title}'"
+    )
```

Comment:
Don't let Ruff use `()` to break a single string into multiple lines, there are better ways